### PR TITLE
A11Y-2: Add label to language dropdown

### PR DESCRIPTION
**Pull request by ebeastlake**
This PR adds a translated `aria-label` to the `select` element that allows the user to choose the page language in the studio.code.org footer.

Screen Shot 2022-12-14 at 10 45 42 AM

Screen Shot 2022-12-14 at 10 44 55 AM

## Links

Jira ticket: https://codedotorg.atlassian.net/browse/A11Y-2

## Testing story

Tested locally with VoiceOver, see screenshots above.

### CODE REVIEW

File: dashboard/app/views/layouts/_footer.html.haml [hunk 1]
```diff
@@ -38,6 +38,6 @@
         -# user input or to persist data without also updating it to enforce UTF-8
         = form_tag(locale_url, method: :post, id: 'localeForm', style: 'margin-bottom: 0px;', enforce_utf8: false) do
           = hidden_field_tag :user_return_to, request.url
-          = select_tag :locale, options_for_select(options_for_locale_select, locale), onchange: 'this.form.submit();'
+          = select_tag :locale, options_for_select(options_for_locale_select, locale), { onchange: 'this.form.submit();', 'aria-label':  I18n.t('footer_locale_select') }
```
Comment (breville) [inline dashboard/app/views/layouts/_footer.html.haml]: The rest of this file appears to just use `t()`, so this should probably remain consistent.

Comment (breville) [reply dashboard/app/views/layouts/_footer.html.haml]: And should be a `footer.` style reference to a string near [here](https://github.com/code-dot-org/code-dot-org/blob/407599dea267d83c2fcbd2ae5e90244f0990d73c/dashboard/config/locales/en.yml#L600).

Comment (ebeastlake) [reply dashboard/app/views/layouts/_footer.html.haml]: I noticed that based on your first comment. Now I'm a little confused though, does editing [this Google sheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0) no longer apply? Or does the `dashboard` still source information from there, and there's just some massaging that happens in between?

Comment (ebeastlake) [reply dashboard/app/views/layouts/_footer.html.haml]: Seems like these get added manually? https://github.com/code-dot-org/code-dot-org/commits/staging/dashboard/config/locales/en.yml

Comment (breville) [reply dashboard/app/views/layouts/_footer.html.haml]: That google sheet is just for pegasus code.  For dashboard, you can add the new string to `dashboard/config/locales/en.yml` in this PR.

**Review by megcrenshaw — APPROVED** [review]
HURRAY!!

**Pull request merged**